Assets/AnkrSDK/Examples/Scripts/EventListenerExample/EventListenerExample.cs

```diff
@@ -45,6 +45,21 @@ public async UniTaskVoid SubscribeWithTopics()
 				(TransferEventDTO t) => ReceiveEvent(t)
 			);
 		}
+		
+		// If you know only topic name then you can use EventFilterRequest
+		public async UniTaskVoid SubscribeWithRequest()
+		{
+			var filtersRequest = new EventFilterRequest<TransferEventDTO>();
+			filtersRequest.SetFromBlock(BlockParameter.CreateLatest());
```

Comment:
Should we also use Earliest here?

Reply:
Actualy it's no need here. FromBlocks works only for request past events